protoc-gen-pbandk/lib/src/jvmTest/kotlin/CodeGeneratorTest.kt

```diff
@@ -30,8 +30,10 @@ class CodeGeneratorTest {
         // Ensure classes and fields were generated successfully
         val message1Clazz = result.classLoader.loadClass("foobar.Message1").kotlin
         val message2Clazz = result.classLoader.loadClass("foobar.Message2").kotlin
+        val message3Clazz = result.classLoader.loadClass("foobar.Variable").kotlin
```

Comment:
Instead of adding this to `simple.proto` and the existing unit tests, please create a separate `.proto` file with the `enum` definition and add a new unit test in this file that loads the new proto file. A single unit test shouldn't validate more than one bug/requirement.

Reply:
I refactored the unit test as I had suggested.